open_gpt/cli/commands/deploy.py

```diff
@@ -0,0 +1,122 @@
+from cleo.commands.command import Command
+from cleo.helpers import argument, option
+
+
+class DeployCommand(Command):
+    name = "deploy"
+
+    description = "Start to deploy a model to JCloud or AWS."
+
+    arguments = [argument("model_name", "The name of the model to serve.")]
+    options = [
+        option(
+            'grpc_port',
+            None,
+            'The gRPC port to serve the model on.',
+            flag=False,
+            default=51001,
+        ),
+        option(
+            'http_port',
+            None,
+            'The HTTP port to serve the model on.',
+            flag=False,
+            default=51002,
+        ),
+        option(
+            'name',
+            None,
+            'The name of the deployment.',
+            flag=False,
+            default='open-gpt-deployment',
+        ),
+        option(
+            'cloud',
+            None,
+            'The cloud to deploy the model on.',
+            flag=False,
+            default='jcloud',
+        ),
+        option('enable_cors', None, 'Enable CORS.', flag=True),
+        option(
+            'precision', None, 'The precision of the model.', flag=False, default='fp16'
+        ),
+        option(
+            'adapter_name_or_path', None, 'The name or path of the adapter checkpoint.'
+        ),
+        option(
+            'device_map',
+            None,
+            'The device map of the model.',
+            flag=False,
+            default='balanced',
+        ),
+        option(
+            "replicas", "r", "The number of replicas to serve.", flag=False, default=1
+        ),
+        option(
+            "instance_type",
+            None,
+            "The instance used to deploy model.",
+            flag=False,
+            default='G3',
+        ),
+        option("config", None, "The config YAML used to deploy.", flag=False),
+    ]
+
+    help = """\
+    This command allows you to deploy a model on JCloud or AWS.
+
+    To start a model deploying, you can run:
+
+        <comment>opengpt deploy facebook/llama-7b</comment>"""
+
+    def handle(self) -> int:
+        if self.option('cloud') == 'jcloud':
+            from open_gpt.config import settings
+            from open_gpt.serve.flow import SimpleFlow
+
+            if self.option('config') is None:
+                flow = SimpleFlow(
+                    name=self.option('name'),
```

Comment:
Basically, I don't like the idea of template rendering. It is overkill. We already have defined the flow via python. We can directly export the python flow to YAML.

Reply:
@jemmyshin